chart/templates/deployment.yaml

```
@@ -86,6 +86,31 @@ spec:
         - name: REDIRECT_LOGS_DOCKER
           value: "true"
         {{- end }}
+        {{- if (default false .Values.rds.useEnvVars) }}
+        {{- if hasKey .Values.env "rdsHost" }}
+        - name: PGHOST
+          value: {{ .Values.env.rdsHost | quote }}
+        {{- end }}
+        {{- if hasKey .Values.env "rdsUser" }}
+        - name: PGUSER
+          value: {{ .Values.env.rdsUser | quote }}
+        {{- end }}
+        {{- if and (hasKey .Values.env "rdsPassword") (hasKey .Values.env.rdsPassword "fromSecret") }}
+        - name: PGPASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.env.rdsPassword.fromSecret }}
+              key: password
+        {{- end }}
+        {{- if hasKey .Values.env "rdsPort" }}
+        - name: PGPORT
+          value: {{ .Values.env.rdsPort | quote }}
```

Similar changes everywhere.
```suggestion
          value: {{ .Values.env.rds.port | quote }}
```